SkietbaanBE/SkietbaanBE/Controllers/FeaturesController.cs

```diff
@@ -32,5 +32,34 @@ public User Search(string username)
             }
             return null;
         }
+
+        //// PUT: api/User/Update
+        [HttpPut]
+        [ActionName("Update")]
+        public async Task<IActionResult> PutUserMember([FromBody] User user)
+        {
+            if (user.Username == null)
+            {
+                return new BadRequestObjectResult("No empty fields allowed");
+            }
+            else
+            {
+                User dbUser = _context.Users.Where(u => u.Username == user.Username)
```

Comment:
Use a `var` here, it makes code updates easier

Reply:
okay did that

Reply:
did that